### DISCUSSION TITLE
extending Pfelk integrating soc tools and alerting?

### DISCUSSION THREAD

**Discussion by tbaror** [pfelk/pfelk#340] [Ideas]
Hello,
Is there plan of extending Pfelk integrating soc tools ?
like [TheHive ](http://thehive-project.org/), Cortex and [Misp ](https://www.misp-project.org/). alerting using [Elastalert](https://elastalert.readthedocs.io/en/latest/)
You can combine all 3 products to automate security events analysis, and this will give a boost to this product

**Comment by a3ilson** [comment]
Haven't looked into this but not likely to to merge.
